packages/changed-elements-react/src/NamedVersionSelector/NamedVersionSelector.css

```diff
@@ -103,11 +103,10 @@
     }
 
     ._cer_v1_named-version-entry {
-      min-height: min-content;
+      min-height: calc(3 * var(--iui-size-xl));
```

Comment:
Seems kind of "magic number"ish to me to have this mult here. Why was this necessary? Is there a better value that can be used here? Wouldn't this break if somehow the component that uses this becomes bigger than this forced value?